exercises/convertTemperature/src/utils/util.js

```diff
@@ -0,0 +1,21 @@
+import {
+  SPACE,
+  TEMPERATURE_INPUT,
+  CONVERT_TEMPERATURE_CONTAINER,
+  MESSAGE_ERROR_NOT_INTEGER,
+} from '../constants.js';
+export const convertedTemperature = (fahrenheitTextList) => {
+  if (!/^[0-9\s]*$/.test(fahrenheitTextList)) {
+    window.alert(MESSAGE_ERROR_NOT_INTEGER);
+  }
+  // Sanitize it
+  const fahrenheitList = fahrenheitTextList.trim().split(SPACE);
+  var convertedTemperatures = '';
+  fahrenheitList.forEach((fahrenheit) => {
+    // Do the math
+    const celsius = ((fahrenheit - 32) * 5) / 9;
+    convertedTemperatures = convertedTemperatures + ' ' + celsius.toFixed(2);
+  });
+  return convertedTemperatures;
```

Comment:
Don't use var. It is not recommeded anymore. Use let or const.

Tip: use `map` and `join` functions. This will avoid the empty space at the beginning of the returned convertedTemperatures. 

 ```suggestion
  const convertedTemperatures = fahrenheitList.map((fahrenheit) => {
    const celsius = ((fahrenheit - 32) * 5) / 9;
    return celsius.toFixed(2);
  }).join(' ');
```